sklearn/metrics/pairwise.py

```diff
@@ -550,12 +550,15 @@ def haversine_distances(X, Y=None):
     (Buenos Aires, Argentina) and the Charles de Gaulle Airport (Paris, France)
 
     >>> from sklearn.metrics.pairwise import haversine_distances
+    >>> from math import radians
     >>> bsas = [-34.83333, -58.5166646]
```

Comment:
I would be okay with starting out with radians:

```py
bsas_in_rad = [-0.6079563, -1.0213085]
paris_in_rad = [0.8553578, 0.0443042]
```

Reply:
Actually I prefer the current way, it's easier to find lat and long and they can be more easily confirmed?